coverage.sbt

```diff
@@ -1,6 +1,10 @@
 coverageExcludedPackages := Seq(
   "<empty>",
   ".*\\.tezos.Tables",
+  ".*\\directives.EnableCORSDirectives",
```

Comment:
Why does this pr specifically require updates to our coverage report?

Reply:
It must be an old change introduced by the initial versions.

Possibly I needed to start filtering out code that we didn't really expect to be testing, to get a realistic coverage %